src/components/Delegate/StakingOpportunities.tsx

```diff
@@ -1,15 +1,169 @@
 import { ArrowForwardIcon } from '@chakra-ui/icons'
-import { Button, HStack, Stack } from '@chakra-ui/react'
+import {
+  Box,
+  Button,
+  Flex,
+  HStack,
+  Skeleton,
+  SkeletonCircle,
+  Tag,
+  TagLabel
+} from '@chakra-ui/react'
+import { AprTag } from 'plugins/cosmos/components/AprTag/AprTag'
+import { StakingAction } from 'plugins/cosmos/components/modals/Staking/Staking'
+import { useMemo } from 'react'
 import { NavLink } from 'react-router-dom'
+import { Column } from 'react-table'
+import { Amount } from 'components/Amount/Amount'
+import { AssetIcon } from 'components/AssetIcon'
 import { Card } from 'components/Card/Card'
-import { Text } from 'components/Text'
+import { ReactTable } from 'components/ReactTable/ReactTable'
+import { RawText, Text } from 'components/Text'
+import { useModal } from 'context/ModalProvider/ModalProvider'
 
-import { StakingOpportunitiesRow } from './StakingOpportunitiesRow'
+type StakingOpportunity = {
+  id: number
+  validatorName: string
+  apr: number
+  stakedAmount?: string | undefined
+  stakedRewards?: string | undefined
+  fiatRewards?: number | undefined
+}
+
+type StakingOpportunitiesProps = {
+  stakingOpportunities: StakingOpportunity[]
+}
+
+type ValidatorNameProps = {
+  validatorName: string
+}
+
+export const ValidatorNameNotStacking = ({ validatorName }: ValidatorNameProps) => {
+  const isLoaded = true
+
+  return (
+    <Box cursor='pointer'>
+      <Flex alignItems='center' maxWidth='180px' mr={'-20px'}>
+        <SkeletonCircle boxSize='8' isLoaded={isLoaded} mr={4}>
+          <AssetIcon
+            src={'https://assets.coingecko.com/coins/images/16724/small/osmo.png?1632763885'}
+            boxSize='8'
+          />
+        </SkeletonCircle>
+        <Skeleton isLoaded={isLoaded} cursor='pointer'>
+          <RawText fontWeight='bold'>{`${validatorName}`}</RawText>
+        </Skeleton>
+      </Flex>
+    </Box>
+  )
+}
 
-export const StakingOpportunities = () => {
-  // TODO: wire up with real validator data
-  const validators = [{ id: 1, name: 'Cosmos Validator' }]
+export const ValidatorNameStacking = ({ validatorName }: ValidatorNameProps) => {
```

Comment:
Not a fan of duplicating this for the `staking` / "not staking" state as both components are effectively the same and have the same API, the only difference is one JSX node is different between the two.
You can either:
- use `children` to use the different node
- pass a discriminator prop to render the right node

I'd vouch for the second option as it's cleaner, plus the components are small so that won't add much boilerplate